policyengine_us/variables/gov/states/ny/tax/income/credits/college_tuition/ny_college_tuition_credit.py

```diff
@@ -8,11 +8,11 @@ class ny_college_tuition_credit(Variable):
     unit = USD
     definition_period = YEAR
     reference = "https://www.nysenate.gov/legislation/laws/TAX/606"  # (t)
-    defined_for = StateCode.NY
+    defined_for = "ny_college_tuition_credit_eligible"
 
     def formula(tax_unit, period, parameters):
-        tuition = add(tax_unit, period, ["qualified_tuition_expenses"])
+        tuition = tax_unit("ny_allowable_college_tuition_expenses", period)
         p = parameters(period).gov.states.ny.tax.income.credits.college_tuition
-        amount = p.rate.calc(tuition) * p.applicable_percentage
-        itemizes = tax_unit("ny_itemizes", period)
-        return ~itemizes * amount
+        # Apply a tiered rate structure.
```

Comment:
Looking at the [instructions](https://www.tax.ny.gov/forms/current-forms/it/it272i.htm#:~:text=Enter%20the%20college%20tuition%20credit,than%20your%20allowable%20standard%20deduction.) I see a $400 limit per person:
![image](https://github.com/user-attachments/assets/05aa3ab3-db0a-468e-bab6-6a6a53302d45)

Is this derived mathematically?

Reply:
Yes - the law limits allowable tuition to $10,000, and only up to 4%. $10,000 * 4% = $400.